### ISSUE TITLE
SDL2 doesn't compile on x86_64

### ISSUE THREAD

**Issue by marijnvdwerf** [marijnvdwerf/openrct2-dependencies-android#3]
```
[1/118] Building C object CMakeFiles/SDL2.dir/src/SDL.c.o
[2/118] Building C object CMakeFiles/SDL2.dir/src/SDL_assert.c.o
[3/118] Building C object CMakeFiles/SDL2.dir/src/SDL_error.c.o
[4/118] Building C object CMakeFiles/SDL2.dir/src/SDL_hints.c.o
[5/118] Building C object CMakeFiles/SDL2.dir/src/SDL_log.c.o
[6/118] Building C object CMakeFiles/SDL2.dir/src/atomic/SDL_atomic.c.o
[7/118] Building C object CMakeFiles/SDL2.dir/src/atomic/SDL_spinlock.c.o
[8/118] Building C object CMakeFiles/SDL2.dir/src/audio/SDL_audio.c.o
[9/118] Building C object CMakeFiles/SDL2.dir/src/audio/SDL_audiocvt.c.o
[10/118] Building C object CMakeFiles/SDL2.dir/src/audio/SDL_audiodev.c.o
[11/118] Building C object CMakeFiles/SDL2.dir/src/audio/SDL_audiotypecvt.c.o
[12/118] Building C object CMakeFiles/SDL2.dir/src/audio/SDL_mixer.c.o
[13/118] Building C object CMakeFiles/SDL2.dir/src/audio/SDL_wave.c.o
[14/118] Building C object CMakeFiles/SDL2.dir/src/cpuinfo/SDL_cpuinfo.c.o
FAILED: /usr/local/opt/android-sdk/ndk-bundle/toolchains/x86_64-4.9/prebuilt/darwin-x86_64/bin/x86_64-linux-android-gcc  --sysroot=/usr/local/opt/android-sdk/ndk-bundle/platforms/android-21/arch-x86_64 -DSDL2_EXPORTS -DUSING_GENERATED_CONFIG_H -Iinclude -I/Users/marijn/deps/build/x86_64/build/sdl/src/sdl_ext/include -g -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wa,--noexecstack -Wformat -Werror=format-security -g -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wa,--noexecstack -Wformat -Werror=format-security  -std=c99 -DGL_GLEXT_PROTOTYPES -D_REENTRANT -mfpmath=387 -msse2 -msse -m3dnow -mmmx -Wshadow -fvisibility=hidden -Wdeclaration-after-statement -Werror=declaration-after-statement -Wall  -O2 -DNDEBUG -O2 -DNDEBUG  -fPIC -MD -MT CMakeFiles/SDL2.dir/src/cpuinfo/SDL_cpuinfo.c.o -MF CMakeFiles/SDL2.dir/src/cpuinfo/SDL_cpuinfo.c.o.d -o CMakeFiles/SDL2.dir/src/cpuinfo/SDL_cpuinfo.c.o   -c /Users/marijn/deps/build/x86_64/build/sdl/src/sdl_ext/src/cpuinfo/SDL_cpuinfo.c
/Users/marijn/deps/build/x86_64/build/sdl/src/sdl_ext/src/cpuinfo/SDL_cpuinfo.c: In function 'CPU_OSSavesYMM':
/Users/marijn/deps/build/x86_64/build/sdl/src/sdl_ext/src/cpuinfo/SDL_cpuinfo.c:250:5: warning: implicit declaration of function 'asm' [-Wimplicit-function-declaration]
     asm(".byte 0x0f, 0x01, 0xd0" : "=a" (a) : "c" (0) : "%edx");
     ^
/Users/marijn/deps/build/x86_64/build/sdl/src/sdl_ext/src/cpuinfo/SDL_cpuinfo.c:250:34: error: expected ')' before ':' token
     asm(".byte 0x0f, 0x01, 0xd0" : "=a" (a) : "c" (0) : "%edx");
                                  ^
ninja: build stopped: subcommand failed.
```

**Comment by LRFLEW** [comment]
http://stackoverflow.com/questions/18574943/gcc-inline-assembler-implicit-function-declaration-on-powerpc-arch

Change the `-std=c99` to `-std=gnu99`

**Comment by marijnvdwerf** [comment]
Fixed by 2c75bfe687f48823695b0edfac30baf59532e319
